translations/en.js

```diff
@@ -45,7 +45,7 @@ export const general = {
 		checkPhone: "Validate phone number",
 		addEmail: "Add an Email",
 		addProId: "Add a professional identifier",
-		deleteAccount: "Cancel this settings",
+		deleteAccount: "Cancel these settings",
```

Comment:
"Cancel these settings"? Ça devrait pas être quelque chose comme "Delete this account" ou quelque chose du genre?

Reply:
J'ai retrouvé ces "Cancel this settings", je ne comprenais pas non plus. Je les ai corrigés mais celle-ci est passée dans la trap.